os/network-config-with-networkd.md

```diff
@@ -17,6 +17,7 @@ Name=enp2s0
 [Network]
 Address=192.168.0.15/24
 Gateway=192.168.0.1
+DNS=192.168.10
```

Comment:
Should that be `192.168.0.10`?

There's also the `10-static.network` a few lines further down which would need the same treatment.

We could also use `DNS=1.2.3.4` to match the other example below's sample value and make it even more obvious the user should fill in the correct one.

Reply:
Oop, nice catch. 

Didn't see that example below, I'll change this to `DNS=1.2.3.4` for consistency's sake.